xilem/examples/http_cats.rs

```diff
@@ -10,6 +10,7 @@
 
 use std::sync::Arc;
 
+use masonry::widget::Padding;
```

Comment:
This should use the re-export from Xilem.

Reply:
Fixed